### ISSUE TITLE
What happens when the screen size is smaller?

### ISSUE THREAD

**Issue by subirranjan1** [michaelnixon/mm-file-support#41]
Hi Annah,

Can you please help us with the design specifications of how the site should look like in smaller devices?What happens to the menu items and the data tables etc?

**Comment by pasquier** [comment]
I do not think that we want to make MoDa mobile friendly...this is a site to be used on the regular sized screen. 
Can we simply indicate a message saying this when the site is accessed from a mobile? 

## Regards, 

## Philippe Pasquier 

Associate Professor, 
Metacreation Lab: Generative Systems 
School of Interactive Arts and Technology, 
Faculty of Communication, Art and Technology, 
Simon Fraser University, Vancouver, Canada. 
Mobile: +1 778-989-1240 | Phone: +1 778-782-8546 
Email: pasquier@sfu.ca | http://www.sfu.ca/pasquier 
Skype: pasquierphilippe | http://www.Metacreation.net 
http://ISEA2015.org/ | http://MusicalMetacreation.org 

## http://MovingStories.ca/| http://MovementComputing.org/ 

----- Original Message -----
